docs/Installing-Lmod-without-root-permissions.rst

```diff
@@ -55,7 +62,8 @@ Lmod
 Dependencies: building Lmod using the steps below requires tcl, tcl-dev(el), make and bzip2
 
 **Step 1**: Download and unpack the latest available Lmod version,
-`Lmod-8.4.tar.bz2`_ at the time of writing.
+`Lmod-8.4.tar.bz2`_ at the time of writing, from the `Lmod GitHub releases page
```

Comment:
@ocaisa This should be updated too, Lmod 8.4 is not latest at time of writing anymore ;)